### PR TITLE
Material-ui based group layout

### PR DESCRIPTION
**Pull request by ga97waqZ**
Adds material-ui specific group layout using `Card`.

### CODE REVIEW

File: packages/material/src/layouts/material-group.layout.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import { groupTester } from './group.layout';
+
+export const MaterializedGroupLayoutRenderer = (props: RendererProps) => {
+    const { uischema, schema, path } = props;
+
+    const group = uischema as GroupLayout;
+    const classNames = JsonForms.stylingRegistry.getAsClassName('group-layout');
+
+    return (
+            <div style={ {marginBottom: '10px' }}>
```
Comment (eneufeld) [inline packages/material/src/layouts/material-group.layout.tsx]: why wrap in an additional div?

File: packages/material/src/layouts/material-group.layout.tsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+                <Card className={classNames}
+                    style={{ padding: '10px 12px 10px 12px',
```
Comment (eneufeld) [inline packages/material/src/layouts/material-group.layout.tsx]: why use custom styling?

File: packages/material/src/layouts/material-group.layout.tsx [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+                        color: 'rgba(0, 0, 0, 0.54)' }}>
+                    {
+                        !_.isEmpty(group.label) ?
+                            <div className={JsonForms.stylingRegistry.getAsClassName
```
Comment (eneufeld) [inline packages/material/src/layouts/material-group.layout.tsx]: why are you not using the `CardHeader` element?

File: packages/material/src/layouts/material-group.layout.tsx [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+                            ('group.label')}
+                            style={{ marginBottom: '15px' }}>
+                                {group.label}
+                            </div> : ''
+                    }
+                    {
+                        renderChildren(
```
Comment (eneufeld) [inline packages/material/src/layouts/material-group.layout.tsx]: Why are you not using `CardContent` ?
also look at the MaterialVerticalLayout how it renders its children

**Comment by eneufeld** [comment]
@ga97waqZ what is the status on the inline comments?
also the index.ts is now invalid.

**Comment by ga97waqZ** [comment]
Sorry tried yesterday night some changes, then noticed that I'm not fit enough anymore and just doing bullshit. I thought I undid all my wrong changes and commited the stuff I was sure about. Must have done more wrong than I thought. I'm currently having a presentation and then a lecture. I will try to fix everything afterwards.

File: packages/material/src/layouts/MaterialGroupLayout.tsx [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+        visible
+    };
+
+    return (
+        <Card className={classNames} style={{marginBottom: '10px' }}>
+            !_.isEmpty(groupLayout.label) ?
+                <CardHeader>
+                    {groupLayout.label}
+                </CardHeader>
+            <MaterialLayoutRenderer {...childProps}/>;
```
Comment (eneufeld) [inline packages/material/src/layouts/MaterialGroupLayout.tsx]: I would wrap this into a CardContent just to mark the children explicitly

File: packages/material/src/layouts/index.ts [hunk 6]
```diff
@@ -1,3 +1,4 @@
 export * from './group.layout';
+export * from './material-group.layout';
```
Comment (eneufeld) [inline packages/material/src/layouts/index.ts]: This doesn't fit with the deletion of the group.layout and the rename of material-group.layout to MaterialGroupLayout

**Comment by edgarmueller** [comment]
@ga97waqZ I've pushed a commit onto your branch in order to fix the problem that the `CardHeader` was not displayed correctly. I've also added a `CardContent`.

File: packages/material/src/layouts/MaterialGroupLayout.tsx [hunk 7]
```diff
@@ -32,14 +30,16 @@ export const MaterializedGroupLayoutRenderer = (props: RendererProps) => {
...
     return (
-        <Card className={classNames} style={{marginBottom: '10px' }}>
-            !_.isEmpty(groupLayout.label) ?
-                <CardHeader>
-                    {groupLayout.label}
-                </CardHeader>
-            <MaterialLayoutRenderer {...childProps}/>;
+        <Card style={{ marginBottom: '10px' }}>
+          {
+            !_.isEmpty(groupLayout.label) &&
```
Comment (eneufeld) [inline packages/material/src/layouts/MaterialGroupLayout.tsx]: the `&&` should be `?` I assume

File: packages/material/src/layouts/MaterialGroupLayout.tsx [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+        schema,
+        path,
+        direction: 'column',
+        visible
+    };
+
+    return (
+        <Card style={{ marginBottom: '10px' }}>
+          {
+            !_.isEmpty(groupLayout.label) &&
```
Comment (eneufeld) [inline packages/material/src/layouts/MaterialGroupLayout.tsx]: `&&` should probably be `?`

**Pull request merged**